readme.md

```diff
@@ -4,15 +4,56 @@ This project contains work in progress templates for unit testing WinUI 3 assemb
 
 You can download the extension from the [Visual Studio marketplace](https://marketplace.visualstudio.com/items?itemName=chingucoding.winuiunittesttemplates).
 
-## Current limitations:
+## Updates
+
+This extension has been updated to work with current stable WinUI and MSTest versions as at 03 October 2023.
+
+TargetFramework and TargetPlatformMinVersion are:
+
+```
+     <TargetFramework>net6.0-windows10.0.19041.0</TargetFramework>
+     <TargetPlatformMinVersion>10.0.17763.0</TargetPlatformMinVersion>
+```
+
+Package references are:
+
+```
+     <PackageReference Include="Microsoft.WindowsAppSDK" Version="1.4.230913002" />
+     <PackageReference Include="Microsoft.Windows.SDK.BuildTools" Version="10.0.22621.756" />
+     <PackageReference Include="MSTest.TestAdapter">
+         <Version>3.1.1</Version>
+     </PackageReference>
+     <PackageReference Include="MSTest.TestFramework">
+         <Version>3.1.1</Version>
+     </PackageReference>
+     <PackageReference Include="Microsoft.TestPlatform.TestHost">
+         <Version>17.7.2</Version>
+         <ExcludeAssets>build</ExcludeAssets>
+     </PackageReference>
+```
+
+## Current issues/limitations:
 
-* Debugger does not work with VS 2022
 * Test runs may occasionally fail with an error of "The active test run was aborted. Reason: Unable to communicate with test host process."
-	* Rebuilding the project or reopening the solution will often fixes this issue.
+	* Rebuilding the project, reopening the solution or reopening VS will often fixes this issue.
 * Debugger with VS 2019 does not register correctly that test run has completed; VS stays in debugging mode.
 * > *VS Test discovery "leaks" two window when running tests, i.e. the templates that host UI will leave a window open when started through the test runner and running the tests will also leave the app open.*
 * This was a problem with WinAppSDK 1.0.0 but does not seem to be present with the latest version of Visual Studio 2022 and WinAppSDK 1.0.3
 
-#### Building
+## Usage
+
+The Unit Test project must compile to a WinExe to support testing operations dispatched to the UI thread.
+
+WinUI 3 does not support referencing a WinExe  project from another WinExe project - you will receive errors on build similar
```

Comment:
Same here, maybe unpackaged instead of WinExe?
Also spacing after the first WinExe.